schemas/com.snowplowanalytics.snowplow/button_click/jsonschema/1-0-0

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a button click event",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "button_click",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "action": {
+      "type": "string",
+      "description": "The action associated with the button click",
+      "minLength": 1,
+      "maxLength": 4096
+    },
+    "elementId": {
+      "description": "The ID of the element clicked if present",
+      "type": [
+        "string",
+        "null"
+      ],
+      "minLength": 1,
+      "maxLength": 4096
+    },
+    "elementClasses": {
+      "description": "An array of class names from the element clicked",
+      "type": [
+        "array",
+        "null"
+      ],
+      "items": {
+        "type": "string"
+      }
+    },
+    "elementName": {
+      "description": "The name of the element if present",
+      "type": [
+        "string",
+        "null"
+      ],
+      "minLength": 1,
+      "maxLength": 4096
+    },
+    "elementContent": {
+      "description": "The content of the element if present and enabled",
```

Comment:
This is basically the label of the button, right? Why not just have a `label` property?

Reply:
I was trying to keep this consistent with `link_click` (and gtm to an extent), but `label` works fine for me too.

Reply:
`elementContent` has been removed in favour of the `label` property.